### ISSUE TITLE
Search filter is not persistent when switching the toggle "Show only PRs"

### ISSUE THREAD

**Issue by culinko** [elmadev/elmaonline-site#542]
This happens on the Profile/Times page. Basically there are 2 separate search filters at the same time, one is for when the toggle is enabled and one is for when the toggle is disabled. Steps to reproduce are the following:

1. Have the toggle "Show only PRs" disabled
2. Enter "Internal 01" into the search field and press enter
3. Enable the toggle "Show only PRs"
4. The results are not being filtered by the "Internal 01" filter, see below screenshot

![image](https://user-images.githubusercontent.com/11700272/156896845-f81d7dcf-f791-4fe1-b2d0-44d06bc4b0ba.png)

5. Continue with the scenario, now enter "Internal 02" into the search field and press enter
6. Disable the toggle "Show only PRs"
7. The results are not being filtered by the "Internal 02" search filter but instead they are being filtered by the "Internal 01" search filter which was applied in step 2, see below screenshot

![image](https://user-images.githubusercontent.com/11700272/156896901-7a720b3f-fee0-4519-99dd-00fd1ce893c3.png)

In conclusion, there is one separate search filter "Internal 01" for when the "Show only PRs" is disabled and another separate search filter "Internal 02" for when the "Show only PRs" is enabled. I think this is not the usual behavior that is to be expected and there should only be one search filter at the time, regardless of the state of the "Show only PRs" toggle.
